models/index.js

```diff
@@ -0,0 +1,31 @@
+import Sequelize from 'sequelize'
+import aboutsModel from './abouts'
+import cookiesModel from './cookies'
+import cookiesTagsModel from './cookiesTags'
+import tagsModel from './tags'
+
+const connection = new Sequelize('cookiesApi', 'cookies', 'C0ok!#$', {
+  host: 'localhost', dialect: 'mysql'
+})
```

Comment:
Yeah this doesn't work on other people's machine

![image](https://user-images.githubusercontent.com/5775716/85067205-4cc20200-b17e-11ea-80f2-ee2641c37522.png)